core/DataTable/Row.php

```diff
@@ -512,6 +495,51 @@ public function sumRow(Row $rowToSum, $enableCopyMetadata = true, $aggregationOp
         }
     }
 
+    protected function getColumnAggregationOperation($columnNameOrId, $aggregationOperations)
+    {
+        // max_actions is a core metric that is generated in ArchiveProcess_Day. Therefore, it can be
+        // present in any data table and is not part of the $aggregationOperations mechanism.
+        if ($columnNameOrId == Metrics::INDEX_MAX_ACTIONS) {
```

Comment:
Can we use strict comparison here?

Reply:
I wouldn't be sure if the index is stored as integer or string in datatables, so changing that might have the potential to break things.